pkgs/build-support/appimage/default.nix

```diff
@@ -46,17 +46,16 @@ rec {
       cd $APPDIR
       exec ./AppRun "$@"
     '';
-  });
+  } // (builtins.removeAttrs args [ "name" "src" "extraPkgs" ]));
```

Comment:
Would `with builtins; attrNames (functionArgs wrapAppImage)` be preferable here?

Reply:
Oh, neat! I think so, no need to maintain two lists and perhaps is clearer re:intent.